src/main/java/org/reactome/release/qa/common/SkipList.java

```diff
@@ -0,0 +1,78 @@
+package org.reactome.release.qa.common;
+
+import org.gk.model.GKInstance;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkipList {
+
+    public List<String> skipList = new ArrayList<>();
```

Comment:
Just curious, why store the db ids as Strings instead of Longs?

Reply:
I guess just because that's the default value from the `Files.readAllLines()` method - I have changed it to use Long values since that does make more sense